index.html

```diff
@@ -3,10 +3,162 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,400;0,700;0,900;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@700;900&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Ili Artes - crafts</title>
   </head>
   <body>
+    <header>
+      <div id="submenu-container">
+        <ul id="submenu">
+          <li class="submenu-item social">
+            <img src="./assets/icons/whiteFB.png" alt="" />
+          </li>
+          <li class="submenu-item social">
+            <img src="./assets/icons/Twitter_bird.png" alt="" />
+          </li>
+          <li class="submenu-item">English</li>
+          <li class="submenu-item">My Page</li>
+          <li class="submenu-item">Logout</li>
+        </ul>
+      </div>
+      <nav>
+        <img
+          id="menu-btn"
+          src="./assets/icons/menu-btn.png"
+          alt="menu button"
+        />
+      </nav>
+      <ul id="menu">
+        <li id="logo-container">
+          <a id="logo" href="./index.html"
+            ><img src="./assets/Mobile/about/Logo.svg" alt=""
+          /></a>
+        </li>
+        <li class="menu-item"><a href="./about.html">About</a></li>
+        <li class="menu-item"><a href="/">Program</a></li>
+        <li class="menu-item"><a href="/">Join</a></li>
+        <li class="menu-item"><a href="/">Sponsor</a></li>
+        <li class="menu-item"><a href="/">News</a></li>
+        <li class="menu-item"><a href="./index.html">CC Campain</a></li>
+      </ul>
+      <div id="logo-container-mobile">
+        <a id="logo-mobile" href="./index.html"
+          ><img src="./assets/Mobile/about/Logo.svg" alt=""
+        /></a>
+      </div>
+      <img src="./assets/icons/menu-close.svg" alt="" />
+    </header>
+    <main>
+      <section id="hero-section">
+        <h3 class="subtitle">Une, pinta, crea.</h3>
+        <h1 class="title">Ili Artes<br /><span>Arte en tus manos!</span></h1>
+        <p>
+          A joyful celebration believing in the value of openness and sharing,
+          creating a positive change with people from over 80 countries is
+          taking place in October, in Korea.
+        </p>
+        <p>2015.10.15(THU) ~ 16(FRI)</p>
+        <p>@ National Museum of Karea, Art Center Nabi and more</p>
+      </section>
```

Comment:
You did great work on implementing the change required by the previous reviewer, but there is little thing you should perform on to go to the next steps:
Kindly I suggest you give some margin or padding, why not center the contents on the container to match the design.

> ### The design
>
><img width="1080" alt="Screenshot 2023-08-02 at 10 40 46 PM" src="https://github.com/Estete9/IliArtes/assets/91237525/943313d8-9279-4c0f-a7d3-3302aeea7550">
>
> ### Your design
>
><img width="1080" alt="Screenshot 2023-08-02 at 10 41 32 PM" src="https://github.com/Estete9/IliArtes/assets/91237525/1aa25388-d504-4d7f-a1e7-783e90ac92e3">